python/foglamp/services/core/scheduler/scheduler.py

```diff
@@ -17,18 +17,19 @@
 import subprocess
 import signal
 from typing import List
-from foglamp.common.configuration_manager import ConfigurationManager
+
 from foglamp.common import logger
+from foglamp.common import utils as common_utils
 from foglamp.common.audit_logger import AuditLogger
-from foglamp.services.core.scheduler.entities import *
-from foglamp.services.core.scheduler.exceptions import *
 from foglamp.common.storage_client.exceptions import *
 from foglamp.common.storage_client.payload_builder import PayloadBuilder
 from foglamp.common.storage_client.storage_client import StorageClientAsync
+from foglamp.common.configuration_manager import ConfigurationManager
+from foglamp.services.core.scheduler.entities import *
+from foglamp.services.core.scheduler.exceptions import *
 from foglamp.services.core.service_registry.service_registry import ServiceRegistry
 from foglamp.services.core.service_registry import exceptions as service_registry_exceptions
 from foglamp.services.common import utils
```

Comment:
remove this import as as you already imported now

Reply:
Nope. This is **foglamp.services.common.utils** and the already imported is **foglamp.common.utils**!

Reply:
was not it better, if we have added  `local_timestamp` def to services/common/utils itself (as requested in JIRA), as this module also belongs to services package services/core/scheduler/scheduler.py and avoid extra import

Reply:
FOGL-2432 (for services) says that fix for this ticket fixes FOGL-2433 (for plugins) also. Since services and plugins are two separate folders at the same level in hierarchy, therefore it makes sense that the common fix to both should come above of both these folders and therefore foglamp.common.utils is the ideal home for common methods.

Reply:
Also, scheduler starts (and imports) only once during the life time of a foglamp server instance. Hence this additional import cost is minuscule and affordable.

Reply:
No we don;t want to change plugins import for plugins for now. It will be removed later, but as along as it stay it will be from .plugins path only

Reply:
python/foglamp/tasks/statistics/statistics_history.py also use the plugin path. We should change that with test file (2 more file changes only, from initial commit) so that `from foglamp.common import utils as common_utils` import becomes more reasonable;

Reply:
done